tracer/test/test-applications/integrations/Samples.OracleMDA/Samples.OracleMDA.csproj

```diff
@@ -1,8 +1,8 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <ApiVersion Condition="'$(ApiVersion)' == ''">19.10.1</ApiVersion>
-    <TargetFrameworks>net462</TargetFrameworks>
+    <ApiVersion Condition="'$(ApiVersion)' == ''">5.0.0</ApiVersion>
+    <TargetFrameworks>net462;netcoreapp3.0;netcoreapp3.1;net5.0;net6.0;net7.0;net8.0</TargetFrameworks>
```

Comment:
netcoreapp 2.1 was generating an error, so I didn't include it

Reply:
What error? 🤔 We probably don't want to remove it given the previous sample worked? Maybe it needs conditional packages added?

Reply:
From what I can see on the NuGet page at some point they swapped to `.NET Standard 2.1` (looks like v3+) so I think we may need to add conditional package references based on the version of the NuGet